app/assets/javascripts/discourse/tests/acceptance/category-edit-security-test.js

```diff
@@ -1,104 +1,174 @@
-import { queryAll } from "discourse/tests/helpers/qunit-helpers";
 import { click, visit } from "@ember/test-helpers";
 import { test } from "qunit";
 import selectKit from "discourse/tests/helpers/select-kit-helper";
 import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import I18n from "I18n";
 
 acceptance("Category Edit - security", function (needs) {
   needs.user();
 
   test("default", async function (assert) {
     await visit("/c/bug/edit/security");
 
-    const $firstItem = queryAll(".permission-list li:eq(0)");
-
-    const badgeName = $firstItem.find(".badge-group").text();
+    const firstRow = find(".row-body").first();
```

Comment:
The global `find` should not be used anymore.  `queryAll` is the correct way which I see you've removed here - was it causing problems?